scripts/create-release-branch.js

```diff
@@ -95,6 +97,11 @@ shell.exec('git add package-lock.json');
 // Add change to lerna.json
 shell.exec('git add lerna.json');
 
+// If it is a beta release, add all files
```

Comment:
dumb q, why do we want to add all files?

Reply:
since we are going to update the references (e.g. from core to core-bundle in .ts & .json)in the beta we release.